end-to-end-tests/cypress/integration/benefits-and-risks/equalities-impact-assessment.js

```diff
@@ -0,0 +1,51 @@
+/// <reference types ='Cypress'/>
+
+describe('Tests to check equalities impact assessment form', () => {
+    afterEach(() => {
+        cy.storeSessionData();
+    });
+
+    beforeEach(function () {
+        cy.visit(`${Cypress.env('url')}/project/10002121/benefits`);
+    });
+
+    it('Should save yes', () => {
+        cy.get('a[data-test=equalities-impact-assessment]').click();
```

Comment:
You have the character 'a' in the parameter, is that a typo?

Reply:
No - there were two elements with this data attribute and the easiest way to distinguish was to look for an anchor element with this data attribute.